docs/descriptor/packages.rst

```diff
@@ -67,35 +64,72 @@ image you should define repository in a following way:
 
 .. _repo_odcs:
 
-ODCS
+
+URL
 ^^^^
-This way is instructs `ODCS <https://pagure.io/odcs>`_ to generate on demand pulp repositories.
-To use ODCS define repository section in following way:
+This approach enables you to download a yum repository file and corresponding GPG key. To do it, define
+repositories section in a way of:
 
 .. code:: yaml
 
     packages:
         repositories:
-            - name: Extras
-              odcs:
-                  repository: rhel-7-extras-rpm
+            - name: foo
+	      url:
+	        repository: https://web.example/foo.repo
+                gpg: https://web.exmaple/foo.gpg
+
+
+.. _repo_contentsets:
+
 
+Content sets
+^^^^^^^^^^^^
+Content sets are tightly integrated to OSBS style of defining repositories in ``content_sets.yml`` file.
+For local Docker based build these repositories are ignored similarly to Plain repository types and
+we expect repository definitions to be available inside image. See
+`upstream docs <https://osbs.readthedocs.io/en/latest/users.html#content-sets>`_ for more details about
+content sets.
 
 .. note::
+   Behavior of Content sets repositories is changed when running in :ref:`Red Hat Environment<redhat_env>`.
 
-   Only on demand pulp ODCS repositories are supported now.
+There are two possibilities how to define Content sets type of repository:
 
+Embedded
+""""""""
+In this approach content sets are embedded inside image descriptor under the ``content_sets`` key.
 
-URL
-^^^^
-This approach enables you to download a yum repository file and corresponding GPG key. To do it, define
-repositories section in a way of:
+.. code:: yaml
+
+    packages:
+        repositories:
+            - name: content_sets
```

Comment:
Should we highlight name here? Is it required?

Reply:
no its not. it can be any name